src/app/static/src/js/components/runReport/runReport.vue

```diff
@@ -29,17 +29,29 @@
                 <div v-for="(options, name) in metadata.instances" v-if="options.length > 1" class="form-group row">
                     <label :for="name" class="col-sm-2 col-form-label text-right">Database "{{ name }}"</label>
                     <div class="col-sm-6">
-                        <select class="form-control" :id="name" v-model="selectedInstances[name]">
-                            <option v-for="option in options" :value="option">
+                        <select class="form-control" :id="name" :ref="`instance-${name}`" @change="changeInstance(name)">
+                            <option v-for="option in options" :value="option"
+                                    :selected="option === selectedInstances[name] ? 'selected' : undefined">
```

Comment:
Hmm, it didn't seem to update when the value changed - although it set it correctly initially. I discovered the issue when trying to add a (deep) watch to clear the status on change.

Reply:
Ok, that sounds very plausible. I think the actual problem is in `mounted`, where using `this.$set(this.selectedInstances, key, instances[key][0])` rather than `this.selectedInstances[key] = instances[key][0]` seems to fix watching. And the watcher itself needs `this.$data.runningStatus`. But this is all rather difficult to debug so let's go with whatever works!